File: ros_tutor/scripts/pub_custom.py
#!/usr/bin/env python

import rospy
from ros_tutor.msg import custom 


def main():
	rospy.init_node("pub_custom_msg")
	rospy.loginfo("Start custom msg publisher")
	pub = rospy.Publisher("/custom_msg",custom,queue_size=1)
	data = custom()
	while not rospy.is_shutdown():
		data.string_msg = "HI"
		data.int64_msg = -1524
		data.uint64_msg = 142
		data.float64_msg = 54.25
		data.bool_msg = True

		pub.publish(data)

if __name__ == '__main__':
	main()

File: ros_tutor/src/pub_custom.cpp
#include <ros/ros.h>
#include <ros_tutor/custom.h>

int main(int argc, char **argv)
{
	ros::init(argc, argv, "pub_custom");

	ros::NodeHandle n;

	ros::Publisher pub = n.advertise<ros_tutor::custom>("pub_custom" , 1000);

	ros::Rate loop_rate(100);

	ROS_INFO("Starting Publisher Custom MSG");

	while (ros::ok())
	{
		ros_tutor::custom data;

		data.string_msg = "stringing";
		data.int64_msg = -323;
		data.uint64_msg = 23;
		data.float64_msg = -25.52;
		data.bool_msg = true;

		pub.publish(data);

		ros::spinOnce();

		loop_rate.sleep();
	}
}